src/components/NewStudentGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Box, FormLabel, Input, Button } from '@chakra-ui/react'
+import firebase from 'firebase'
+import { useHistory } from 'react-router-dom'
+
+type NewStudentGroupProps = {
+  studentGroupsRef: firebase.firestore.CollectionReference
+}
+
+const NewStudentGroup: React.FC<NewStudentGroupProps> = ({ studentGroupsRef }) => {
+  const [studentGroupNameInput, setStudentGroupNameInput] = React.useState('')
+
+  const history = useHistory()
+
+  async function submitHandler(e: React.BaseSyntheticEvent | undefined) {
```

Comment:
why would this ever be undefined?

Reply:
I don't know.  I'll remove the "| undefined".  I don't remember why I did that back when I wrote that function (I copied it from another component into this new component)